app/client/src/utils/Analytics/smartlook.ts

```
@@ -0,0 +1,19 @@
+import smartlookClient from "smartlook-client";
+import { getAppsmithConfigs } from "ee/configs";
+
+class SmartlookUtil {
+  static initialised: boolean = false;
+
+  public static init() {
+    const {
+      smartLook: { enabled, id },
+    } = getAppsmithConfigs();
+
+    if (enabled && !this.initialised) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)